test/testdata/unpacked_fixed_request.json

```diff
@@ -0,0 +1,159 @@
+{
+  "format": "metro",
```

Comment:
Is format here just for the print endpoint? What if we made it a query param?

Reply:
This would also work for the convert endpoint as well. You're right though, it makes a lot more sense to move format to the query param.